spec/test_app/db/schema.rb

```diff
@@ -37,6 +37,15 @@
   add_index "basechurch_api_keys", ["access_token"], name: "index_basechurch_api_keys_on_access_token", unique: true
   add_index "basechurch_api_keys", ["user_id"], name: "index_basechurch_api_keys_on_user_id"
 
+  create_table "basechurch_attachments", force: true do |t|
+    t.integer  "element_id"
+    t.string   "element_type"
+    t.string   "element_key"
+    t.string   "url"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.